### DISCUSSION TITLE
n-gon parkour level pt. 2

### DISCUSSION THREAD

**Discussion by tbxyd** [landgreen/n-gon#363] [Show and tell]
I made another really hard parkour level, but this one is even more annoying
Everything is possible, just some stuff is really hard
go to [this website](https://c-rxxp-y.GitHub.io/n-gon-enhanced) to play the level, or go to [this website](https://n-gon-enhanced.vercel.app) to play my mod

**all jumps below are in chronological order**

no prize for beating this one, its just really hard

**note:** the link for the main modded version has been changed to https://n-gon-enhanced.vercel.app
if you decide to use https://c-rxxp-y.github.io/n-gon-enhanced you will be taken to the parkour version

Jump 1: Low Jump
Difficulty: 8/10
Description: A high jump landing on a corner clip with a 3 frame jump window (just tap the button), allows the character to jump shorter than what is possible by basic movement. Probably the most annoying jump.

Jump 2: Clip Up
Difficulty: 1/10
Description: With certain height jumps, which would be impossible normally, holding down the jump button allows your character to clip up to the block above.

Jump 3: Corner Clip + Momentum Cancel
Difficulty: 6/10
Description: Basic bread and butter corner clip followed by a momentum cancel where you crouch and then instantly jump to rid yourself of almost all momentum in all directions.

Jump 4: Midair Crouch
Difficulty: 4/10
Description: Same height hole as player allows player to crouch quickly midair, though it is not necessary and can be done without crouching.

Jump 5: Anti-Crouch
Difficulty: 2/10
Description: Another roundabout to reset your crouch.

Jump 6: 1 Pixel Forward Corner Clip
Difficulty: 4/10
Description: Easier variation of the corner clip. Easier setup because of the limited space.

Jump 7: Momentum Storage
Difficulty: 4/10
Description: Landing on this block at the corner allows you to decrease the rate at which you are falling while still keeping any horizontal momentum.

Jump 8: 1 Pixel to 1 Pixel
Difficulty: 1/10
Description: Just a normal jump.

Jump 9: Backwards Corner Clip Roundabout
Difficulty: 7/10
Description: Same thing as a forward corner clip, but with harder timing and a roundabout at the top.

Jump 10: Quad Ceiling Clip
Difficulty: 10/10
Description: Frame perfect input, extremely difficult, almost impossible to do consistently. Involves clipping through all 4 ceiling blocks without triggering a momentum change. Hardest jump in the level.

Jump 11: DOUBLE JUMP!!!
Difficulty: 2/10 with setup, 10/10 without.
Description: With precise setups after a corner clip (ceiling in this case down to the floor), bouncing off of a perfectly placed wall within another wall to reset your jump and gives the ability to jump again (only if you hold). Mix of corner clip, clip up, and ceiling clip.

Jump 12: Long Corner Clip
Difficulty: 5/10
Description: Basic corner clip, covers long distance fast. Nothing special.

Jump 13: Lock Box
Difficulty: 0/10
Description: Cool tech I found, just wanted to show off, allows characters to enter a room by clicking jump twice and clipping through a wall but does not allow them to exit the room after.

**Comment by AnnonymousNerd87** [comment]
wow, I'll see this later. Also I'll get back to you about the skin tech

**Comment by AnnonymousNerd87** [comment]
Okay I have the skin tech idea: antimatter: 2x damage, lose health and gain
energy when firing

uses a modified mass-energy skin with darker colors

if you've ever played Nova Drift it should be like the super tech
antimatter rounds

**Comment by AnnonymousNerd87** [comment]
the health lost is 5% + 5 health, cannot reduce your health below 1, can
only trigger up to 3 times per second.

if possible, instead of 2x damage it's based on health lost, maybe 10x
health lost as damage or smth

**Comment by tbxyd** [comment]
I really like this idea, I'll start working on it. yesterday I put out a new skin where jump height is based on energy level (I'm really proud of how it looks too, that shit took 2 hours to make ☠)

**Comment by tbxyd** [comment]
I'm thinking light green outline with dark green head, same color limbs but the joint color could be dark green or smth. green obviously bc of health and since no other skin uses that color (that im aware of)

**Comment by AnnonymousNerd87** [comment]
ye, sounds good

**Comment by AnnonymousNerd87** [comment]
Given the fact it took me like 2 1/2 weeks to come up with the idea

**Comment by tbxyd** [comment]
yeah lol, I like it tho. in an hour or so I'll send a screenie of how it looks. i'm not gonna add it to the game right away as I obviously have to test it, but I'm excited for how its going to turn out

**Comment by AnnonymousNerd87** [comment]
of course

**Comment by tbxyd** [comment]
do you have a discord? it'd be way easier to talk if we added eachother

**Comment by AnnonymousNerd87** [comment]
and make sure not to waste your break on a random idea for a hobby!

**Reply by tbxyd** [reply]
...shit... I was planning on that ☠

**Comment by AnnonymousNerd87** [comment]
I have discord, but not on my school chromebook for obvious reasons, and I
will be away from both email and discord over break

**Reply by tbxyd** [reply]
ah ok. lmk when you get back. I will also be offline for most of break, might do some work on upcoming updates 👀

**Comment by tbxyd** [comment]
some parts of the skin are going to change color based on what your health level is at
for example full health mostly green, less health less green

**Comment by AnnonymousNerd87** [comment]
sounds cool

**Comment by tbxyd** [comment]
[antimatter_skin_showcase.webm](https://github.com/user-attachments/assets/b5845c67-4354-4e1c-b3aa-867930d0cf0e)
showcase of how the skin looks, obviously I suck at the game
I went through a couple levels to show what different values of health looks like
low = grayer high = greener

**Comment by AnnonymousNerd87** [comment]
maybe if the head was dark grey/gray and the border was the green but the
border is thicker? I feel like the green doesn't communicate "antimatter"
to me as much

**Comment by AnnonymousNerd87** [comment]
so the border has the hp color thing and the head is dark

**Comment by AnnonymousNerd87** [comment]
I'll send u a pic

**Comment by tbxyd** [comment]
Yeah Ill swap colors around. Just started work tho so Ill mess with it later


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
[image: Screenshot 2024-12-17 3.10.20 PM.png]

**Comment by AnnonymousNerd87** [comment]
https://drive.google.com/file/d/1-JDVjTO65q8SDczbMQBpJ3u1_34SZEJh/view?usp=sharing
this should work, github didn't upload my prev. image

**Comment by tbxyd** [comment]
[Screen recording 2024-12-18 11.55.24 AM.webm](https://github.com/user-attachments/assets/ec73ddfd-ab66-4ee3-ac91-ed4b7e6854eb)
lmk how this looks

**Comment by AnnonymousNerd87** [comment]
That's awesome, great job

**Comment by AnnonymousNerd87** [comment]
has a great antimatter feel, now you just have to code the ability

**Comment by tbxyd** [comment]
yup, that wont be too hard. what did you mean by 5% - 5 health? like you lose 5% of health per shot?

**Comment by AnnonymousNerd87** [comment]
you lose 5% of your current health and 5 flat health (so if you have really
low hp it doesn't make you lose 0 health), this can trigger up to 3 times
per second (so if you're using nailgun you don't instantly burn down to 1
hp), this damage can't reduce your hp below 1

**Comment by AnnonymousNerd87** [comment]
yes, 5% per shot

**Comment by tbxyd** [comment]
since its taking health, would it be a good idea to also give a boost in max health? doesn't have to be a lot, even just +20 to +50 would be good

**Comment by AnnonymousNerd87** [comment]
maybe, we'll see once the life drain is coded, I'm basing it off of Nova
Drift's antimatter rounds upgrade, but that game has natural regeneration
from biomechanical repairs/regeneration

**Comment by AnnonymousNerd87** [comment]
maybe a 20-50% defense boost, similar to zeno's paradox but less

**Comment by AnnonymousNerd87** [comment]
the problem isn't max hp but healing, so it should be either defense or
healing

**Comment by tbxyd** [comment]
maybe healing orbs should just be buffed a little bit so you're not stuck at low health the whole game

**Comment by AnnonymousNerd87** [comment]
sure

**Comment by tbxyd** [comment]
how do you like the name "antimatter turret"
obviously you get to name it, but it was a cool idea I had

**Comment by AnnonymousNerd87** [comment]
because it's a skin tech and not a gun, the name might be misleading. Maybe
"antimatter annihilation" if you don't want just "antimatter"

**Comment by tbxyd** [comment]
well turret is like a building/tower thing that shoots, which would be like m
I feel like antimatter annihilation is a bit too much, maybe something like "antimatter generator" or "antimatter agent"

**Comment by AnnonymousNerd87** [comment]
maybe antimatter cannon if you want, but remember this is a skin tech so
using the n-gon naming conventions "antimatter" or "antimatter
annihilation" would be perfectly fine

**Comment by tbxyd** [comment]
I'll do antimatter

**Comment by tbxyd** [comment]
Basically I've finished the code for it, Ill just have to fix some bugs and fully implement it. There should be an update later today 👀


Sent from my Apple Watch
